docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -67,23 +67,23 @@
 * __File Summary Page__:
     * BAM metrics have been added. <!--PEAR-2231-->
     * The Workflow Completion Date has been removed. <!--PEAR-2131-->
-* The deprecated Sample Type property has been replaced by new sample properties (Tissue Type, Tumor Descriptor, Specimen Type, and Preservation Method) in the __Sample Sheet__. <!--PEAR-2113-->
-* Deprecated properties (Analyte Type ID, Is FFPE, OCT Embedded, and Tumor Code) have been removed from the biospecimen tree in the __Case Summary Page__. <!--PEAR-2306-->
-* As appropriate, deprecated properties (Premature at Birth, Metastasis at Diagnosis Site, Pregnant at Diagnosis, Treatment Anatomic Site) have been removed or replaced in the __Clinical Data Analysis__ tool. Additionally, a new Other Clinical Attribute category has been created. <!--PEAR-2306-->
-* Case counts are no longer displayed for cases with missing Age At Diagnosis values in __Cohort Comparison__. <!--PEAR-2316/SV-2558-->
-* Updated the Repository and Cart to display Sample Sheet and Metadata downloads as separate buttons for better visibility. <!--PEAR-2232-->
-* Minor text and styling improvements. <!--PEAR-452/2298/2341/2323/2253/2290-->
-
 * __OncoMatrix__:
     * By default hide "splice region" consequence mutations.
-    * Support 5-category CNV: in OncoPrint mode, plot a white border around SSM to better distinguish it from CNV when columns are wide enough
+    * Support 5-category CNV: in OncoPrint mode, plot a white border around SSM to better distinguish it from CNV when columns are wide enough.
     * Continuous variable row label menu supports option to edit bar height and color.
-    * Allow user to modify colors from the legend.
+    * Allow users to modify colors from the legend.
 * __Gene Expression Clustering__:
-    * Avoids showing genes with no expression values (due to an issue of ProteinPaint GENCODE v36 data that affects only certain genes; data issue will be fixed at the next release)
+    * Avoids showing genes with no expression values (due to an issue of ProteinPaint GENCODE v36 data that affects only certain genes; data issue will be fixed at the next release).
     * Make zscore transformation a checkbox option in Clustering menu.
 * __Cohort MAF__:
-    * Allowed to sort the table of MAF files and use bar plot to display MAF file size.
+    * Allow users to sort the table of MAF files. 
+    * A bar plot has been added to display the MAF file size.
```

Comment:
@siosonel I split this into two points because it seemed like two separate feature. is that ok?

Reply:
Yes, thanks